src/dom/getancestors.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/**
+ * Returns all ancestors of given DOM node, starting from the top-most (root). Includes the given node itself.
+ *
+ * @param {Element|Text} node DOM node.
+ * @returns {Array.<Node>} Array of given `node` parents.
+ */
+export default function getAncestors( node ) {
+	const nodes = [];
+
+	while ( node ) {
+		nodes.unshift( node );
+		node = node.parentNode;
+	}
+
+	return nodes;
+}
```

Comment:
I'd like to have this consistent, but I must agree... in many, many scenarios you'll want to start searching from the root. And this array should not be long anyway, so an array will be more handy.